deeppavlov/models/morpho_syntax_parser/spacy_lemmatizer.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2019 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List, Optional, Tuple, Union
+
+import spacy
+
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.models.component import Component
+
+
+@register('spacy_lemmatizer')
```

Comment:
maybe make this component `ru_spacy_lemmatizer` and remove language parametrization and eng spacy model from requirements? @vaskonov?